db/access.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+defined('MOODLE_INTERNAL') || die();
+
+$capabilities = array(
+
+    /* Add, edit or remove payment enrol instance. */
+    'enrol/payment:config' => array(
+        'captype' => 'write',
+        'contextlevel' => CONTEXT_COURSE,
+        'archetypes' => array(
+            'manager' => CAP_ALLOW,
+        )
+    ),
+
+    /* Enrol anybody. */
+    'enrol/payment:enrol' => array(
+        'captype' => 'write',
+        'contextlevel' => CONTEXT_COURSE,
+        'archetypes' => array(
+            'manager' => CAP_ALLOW,
+            'editingteacher' => CAP_ALLOW,
+        )
+    ),
+
+    /* Manage enrolments of users. */
+    'enrol/payment:manage' => array(
+        'captype' => 'write',
+        'contextlevel' => CONTEXT_COURSE,
+        'archetypes' => array(
+            'manager' => CAP_ALLOW,
+            'editingteacher' => CAP_ALLOW,
+        )
+    ),
+
+    /* Unenrol anybody (including self) - watch out for data loss. */
+    'enrol/payment:unenrol' => array(
+        'captype' => 'write',
+        'contextlevel' => CONTEXT_COURSE,
+        'archetypes' => array(
+            'manager' => CAP_ALLOW,
+            'editingteacher' => CAP_ALLOW,
+        )
+    ),
+
+    /* Unenrol self - watch out for data loss. */
+    'enrol/payment:unenrolself' => array(
+        'captype' => 'write',
+        'contextlevel' => CONTEXT_COURSE,
+        'archetypes' => array(
+        )
+    ),
+
+);
```

Comment:
Add a newline here. Should be able to config this in your IDE so it autoadds newlines at the end of files